routing/missioncontrol.go

```diff
@@ -121,6 +125,10 @@ type MissionControl struct {
 // MissionControlConfig defines parameters that control mission control
 // behaviour.
 type MissionControlConfig struct {
+	// UpdateEstimator is a callback function that is called whenever the
+	// mission control state is updated.
+	UpdateEstimator func(estimator Estimator)
```

Comment:
what about other config values that are not estimator related such as `MaxMcHistory` and `MinFailureRelaxInterval`? 

I think perhaps we should instead make this more generic: `OnConfigUpdate`?

Reply:
I did the change. But I did not find `MinFailureRelaxInterval` on main cfg. Maybe I'm not looking for on the right place?

Reply:
```
	mcCfg := &routing.MissionControlConfig{
		MaxMcHistory: int(req.Config.MaximumPaymentResults),
		MinFailureRelaxInterval: time.Duration(
			req.Config.MinimumFailureRelaxInterval,
		) * time.Second,
	}
```

Reply:
I mean in the target structure `routerrpc.Config`  there is no MinFailureRelaxInterval

Reply:
oh i see 👍